pkg/controller/machineset/delete_policy.go

```diff
@@ -67,6 +76,34 @@ func oldestDeletePriority(machine *v1alpha1.Machine) deletePriority {
 	return deletePriority(float64(mustDelete) * (1.0 - math.Exp(-d.Seconds()/secondsPerTenDays)))
 }
 
+
+func customDeletePolicy(machine *v1alpha1.Machine) deletePriority {
+	// Iterate through conditions to get if machine is not "Ready"
```

Comment:
Because we always return `mustDelete`, there's no real order to the machines with that value.
That's something we should eventually improve; for now maybe add a comment.

However, maybe reorder these checks already:
- deletion timestamp non zero
- machine has no nodeRef -> still provisioning
- deletion annotation
- node status NotReady
- error status

Reply:
Okay, I'll "split" the NodeReady/Conditions report to make machine selection more granular.